### ISSUE TITLE
[Proposal] Smoother LOD transitions for terrain tiles

### ISSUE THREAD

**Issue by jailln** [iTowns/itowns#2373]
This issue is a **feature proposal**. Feel free to upvote (with :+1: ), comment and provide your use-cases if you're interested by this feature.

## Context
Terrain LOD refinment is currently sharp.

## Description of the proposal
We could implement smoother LODs transitions like what's described in [this blog post](https://cesium.com/blog/2022/10/20/smoother-lod-transitions-in-cesium-for-unreal/). Note that this is the technique used for `3d-tiles-renderer` fading tiles implementation.

**Comment by Desplandis** [comment]
I agree with the feature proposal, it would greatly improve the current render (with a little perf cost, but this is a tradeoff I'm willing to take).

However, we'll first need to clean-up the code for terrain generation, there is a lot of side-effects which could not ease the development of this feature.
